test/AUTH/General/resetAccount.js

```diff
@@ -0,0 +1,28 @@
+import AuthAppActions from "../../../lib/util/AuthApp/actions.js";
+
+const AuthApp = new AuthAppActions();
+
+describe('User should be able to reset', async() => {
+   beforeEach('Navigate to login page for Moodle', async () => {
+      await AuthApp.open();
+      await AuthApp.successfulOpen();
+      await AuthApp.navigateToLogin();
+   }); 
+
+   it('work email', async() => {
+      await AuthApp.resetWorkAccount();
+      await AuthApp.successfulOpen("passwordreset.microsoftonline.com");
```

Comment:
The string that is passed to successfulOpen() should be included in a constant file